File: server/aspnet/sqlserver/nvslonlineApi/nvslonlineApi/Models/News.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace nvslonlineApi.Models
{
    public class News
    {
        public int Id { get; set; }

        public string title { get; set; }

        public string description { get; set; }

        public bool IsHidden { get; set; }

        public DateTime? created { get; set; }

        public string modifiedBy { get; set; }

        public string modifiedByfullName { get; set; }

        public DateTime? modified { get; set; }
    }
}

File: client/angular2/NvslOnlineAngular2App/src/app/shared/models/news.ts
import { INews } from '../interfaces/interfaces';

export class News implements INews{    
    
    constructor (public Id: number, public title: string, public description: string,  public IsHidden: boolean, public created: string) {
        this.Id = Id;
        this.title = title;
        this.description = description;
        this.IsHidden = IsHidden;
        this.created = created;
    } 
}